src/python/pants/backend/python/rules/run_setup_py.py

```diff
@@ -469,25 +472,25 @@ async def determine_setup_kwargs(
     exported_target: ExportedTarget, union_membership: UnionMembership
 ) -> SetupKwargs:
     target = exported_target.target
-    plugin_requests = union_membership.get(SetupKwargsPluginRequest)  # type: ignore[misc]
+    plugin_requests = union_membership.get(SetupKwargsRequest)  # type: ignore[misc]
```

Comment:
Let's expunge the "plugin" terminology in the var names and comments here.

Reply:
Why? This is internal code for us. These _are_ plugin implementations. By definition, they will have been provided outside of `run_setup_py.py` via the Plugin API.

These comments are meant to meant explain what is going on, that we're checking if the Plugin API is used.

End plugin authors won't be exposed to this rule - it's only notes to help us.

Reply:
My thinking is that whether or not the implementations come from a plugin is incidental to this code. The naming implies that they must, which isn't the case. E.g., we could provide a "read from a sibling VERSION file" implementation in the core if we chose to.